core/templates/dev/head/forms/formBuilder.js

```diff
@@ -1358,6 +1378,17 @@ oppia.directive('schemaBasedFloatEditor', [function() {
           }
         };
 
+        if (typeof $scope.uiConfig() !== 'undefined' &&
+            $scope.uiConfig().slider === true) {
+          if (!$scope.localValue) {
+            $scope.localValue = 300.0;
```

Comment:
Don't set a default here; it'll mess up all other float editors elsewhere. If you need to set a default, do that outside this component; this component represents a _general_ editor that's used for all floats in the application.